cli/storage_azure.go

```diff
@@ -0,0 +1,30 @@
+package cli
+
+import (
+	"context"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/kopia/kopia/repo/blob"
+	"github.com/kopia/kopia/repo/blob/azure"
+)
+
+func init() {
+	var azOptions azure.Options
+
+	RegisterStorageConnectFlags(
+		"azure",
+		"an Azure blob storage",
+		func(cmd *kingpin.CmdClause) {
+			cmd.Flag("container", "Name of the Azure container").Required().StringVar(&azOptions.Container)
+			cmd.Flag("account-name", "Azure account name(overrides AZURE_STORAGE_ACCOUNT environment variable)").Required().Envar("AZURE_STORAGE_ACCOUNT").StringVar(&azOptions.AccountName)
```

Comment:
Probably, we should also change `--account-name` flag to `--storage-account`?
@vkamra @julio-lopez